apps/src/netsim/NetSimLobby.js

```diff
@@ -579,9 +581,18 @@ NetSimLobby.prototype.buildShardChoiceList_ = function(
     });
   }
 
-  // If there's only one possible shard, select it by default
```

Comment:
this comment still seems useful for the first conditional block. imo this would be a little clearer if you move lines 584-588 inside the `else if` and restore this comment.